fake-ppud-automation-api/__files/get-offender-X098092.json

```diff
@@ -32,6 +32,7 @@
           "indexOffence": "Blasphemy against coca cola",
           "dateOfIndexOffence": "2008-06-12"
         },
+        "releaseDate": "2005-05-12",
         "releases": [
```

Comment:
Shouldn't the `releases`be removed?

Reply:
And the same on line 75 of this file (GitHub won't let me comment on it because it's too far from changes you've made).